src/main/java/dinkplugin/LevelNotifier.java

```diff
@@ -46,6 +46,7 @@ public void onTick() {
     }
 
     public void attemptNotify() {
+        if (plugin.isSpeedrunWorld()) return;
```

Comment:
I think this check should be in `#handleLevelUp` so `sendMessage` doesn't get set to true